pkg/skaffold/docker/parse.go

```diff
@@ -396,42 +349,30 @@ func retrieveImage(image string, insecureRegistries map[string]bool) (*v1.Config
 	return localDaemon.ConfigFile(context.Background(), image)
 }
 
-func processCopy(value *parser.Node, envs map[string]string) ([]string, error) {
-	var copied []string
+func retrieveWorkingDir(tagged string, insecureRegistries map[string]bool) (string, error) {
```

Comment:
There's a copy of this code in `sync.go`

Reply:
:sweat: I shamelessly copied from there. I think the best place for this will be in the `docker` package.